kim/kim-ldap/src/main/java/edu/arizona/kim/service/impl/UaEdsIdentityService.java

```diff
@@ -141,4 +156,45 @@ public void setPrincipalDao(LdapPrincipalDao principalDao) {
 	public LdapPrincipalDao getPrincipalDao() {
 		return principalDao;
 	}
+
```

Comment:
The correct implementation for the below overridden methods is in Person Service. Throwing an exception here will help us identify other places in the code where we might erroneously trying to access kim tables instead of going through ldap.